libraries/libformula/test/org/pentaho/reporting/libraries/formula/function/userdefined/CsvArrayFunctionTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This program is free software; you can redistribute it and/or modify it under the
+ *  terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ *  Foundation.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License along with this
+ *  program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ *  or from the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Lesser General Public License for more details.
+ *
+ *  Copyright (c) 2006 - 2015 Pentaho Corporation..  All rights reserved.
+ */
+
+package org.pentaho.reporting.libraries.formula.function.userdefined;
+
+import org.pentaho.reporting.libraries.formula.FormulaTestBase;
+
+public class CsvArrayFunctionTest extends FormulaTestBase {
+  public void testDefault() throws Exception {
+    runDefaultTest();
+  }
+
+  public Object[][] createDataTest() {
+    return new Object[][]
+      {
+        { "CSVARRAY(\"1,2,3\"; TRUE(); \",\"; \"'\")" , new String[]{"1","2","3"}},
+        { "CSVARRAY(\"1,2,3\"; FALSE(); \",\"; \"'\")" , new String[]{"1","2","3"}},
+        //{ "CSVARRAY(\"'1,2',3\"; TRUE(); \",\"; \"'\")" , new String[]{"1,2","3"}}
```

Comment:
@tmorgner, why this line is commented? Haven't you forgotten to enable it? The test passes w/o the changes in `CsvArrayFunction` :)